native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/internal/BaseNativeImageOptions.java

```diff
@@ -240,7 +243,13 @@ public BaseNativeImageOptions(String name,
         getDebug().convention(false);
         getFallback().convention(false);
         getVerbose().convention(false);
-        getQuickBuild().convention(false);
+        getQuickBuild().convention(providers.environmentVariable("GRAALVM_QUICK_BUILD").map(env -> {
+            LOGGER.warn("Quick build environment variable is set.");
```

Comment:
In a multi-project build, this is too verbose IMO:

![image](https://user-images.githubusercontent.com/153880/181307332-940c7873-c934-454a-989c-da4e95b071c6.png)

Reply:
Good call, let me fix this.